refinery/data_set_manager/views.py

```diff
@@ -475,8 +476,9 @@ def post(self, request, *args, **kwargs):
             if data_set_ownership_response is not None:
                 return data_set_ownership_response
         try:
-            metadata_file = request.FILES['file']
             title = request.POST.get('title')
+            metadata_file = request.FILES['file']
+            metadata_file.name = request.POST.get('file_name')
```

Comment:
`metadata_file.name` should already contain uploaded file name: https://docs.djangoproject.com/en/1.8/ref/files/uploads/#django.core.files.uploadedfile.UploadedFile.name

Reply:
`metadata_file.name` is: `u'blob'` before this assignment